### ISSUE TITLE
protobuf lead to RuntimeError

### ISSUE THREAD

**Issue by nkchwfree** [yahoojapan/NGT#115]
This is my code.
```
from google.protobuf import descriptor as _descriptor  # protobuf==3.16.0
import ngtpy

ngtpy.create("index", 4, distance_type='Cosine')
index = ngtpy.Index('index')
index.insert([0, 1, 0, 1])
```

Traceback (most recent call last):
  File "_debug_index.py", line 6, in <module>
    index.insert([0, 1, 0, 1])
RuntimeError: /work/NGT/release/NGT/lib/NGT/ObjectRepository.h:

**Comment by masajiro** [comment]
Thank you for your brief code reproducing the issue. I have been investigating the issue.

**Comment by masajiro** [comment]
I have released [v1.14.3](https://pypi.org/project/ngt/1.14.3/#files) that fixed this issue.
